elements/rh-alert/rh-alert.css

```diff
@@ -119,34 +119,33 @@ header ::slotted(:is(h1,h2,h3,h4,h5,h6)) {
 }
 
 #description > ::slotted(*) {
-  margin-top: var(--rh-space-md, 8px);
-  margin-bottom: var(--rh-space-lg, 16px);
+  margin-block: var(--rh-space-md, 8px) var(--rh-space-lg, 16px) !important;
 }
 
 footer.hasActions {
   margin-top: var(--rh-space-lg, 16px);
 }
 
 footer ::slotted([slot="actions"]) {
-  margin-right: var(--actions-item--MarginRight, var(--rh-alert__actions--Gap, 24px)) !important;
```

Comment:
👍

Reply:
yeah i saw that and I was like 

!["ay yay yay yay yay!"](https://media.giphy.com/media/3oz8xETviCTLc93cDC/giphy.gif)